datadog_checks_dev/datadog_checks/dev/tooling/commands/release/stats/stats.py

```diff
@@ -28,6 +30,34 @@ def parse_commit(commit):
     return {'sha': commit.sha, 'title': title, 'url': url, 'teams': ' & '.join(teams), 'next_tag': next_tag}
 
 
+def export_changes_as_csv(changes, filename):
+    with open(filename, "w") as release_csv:
+        echo_info("Exporting csv as {}".format(filename))
+        writer = csv.writer(release_csv)
+
+        # Header
+        writer.writerow(
+            [
+                'PR',
+                'Team',
+                'What RC was it included in?',
+                'Short description',
+                'Severity',
+                'Was this a bug or something else?',
+                'What could we have done differently or what could we do differently to find this bug earlier',
+            ]
+        )
+        for change in changes:
+            writer.writerow(
+                [
+                    change['url'],
+                    change['teams'],
+                    change['next_tag'].split('-')[-1] if '-' in change['next_tag'] else change['next_tag'],
+                    change['title'],
```

Comment:
Is this not lacking the last columns (severity and after)? Or is Python fine and putting blank items automatically?

```suggestion
                    change['url'],
                    change['teams'],
                    change['next_tag'].split('-')[-1] if '-' in change['next_tag'] else change['next_tag'],
                    change['title'],
                    '',
                    '',
                    '',
```